app/src/views/dashboard/Dashboard.jsx

```diff
@@ -6,17 +6,12 @@ import NotFound from "../404/404";
 import PageWrapper from "../../containers/pageWrapper/PageWrapper";
 import GroupLayout from "./groups";
 
+import GroupModal from "../../components/groupModal/GroupModal";
+
 class Dashboard extends React.Component {
   render() {
     return (
-      <div
-        style={{
-          display: "flex",
-          height: "100%",
-          flexGrow: 1,
-          backgroundColor: "#f3f4f8"
-        }}
-      >
+      <div style={{ display: "flex", height: "100%", flexGrow: 1 }}>
```

Comment:
I think will be good to use more classes styles instead inline. @i-barysiuk what do you think?

Reply:
Styling to a separate file is considered good practice, as far as I know